portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminTaxonomyAPI/HeadlessAdminTaxonomySupportDifferentParameters.testcase

```diff
@@ -0,0 +1,123 @@
+@component-name = "portal-headless-frontend-infrastructure"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Headless Frontend Infrastructure";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		TaxonomyVocabularyAPI.setUpGlobalSiteId();
```

Comment:
I'm not sure whether this approach is safe... This way, if anyone needs to use our macro will have to set up the siteId themselves, which may be a bit problematic, maybe it is better if the parts of asking for siteId stays inside the macro. @yanan @Hijky what do you think about it?

Reply:
Agree. What about we pass the siteId as a parameter inside of method in macro? In the codes now we are using "Guest" site, if you'd like to use "Global" site then we can add one more parameter "groupName".